frontend/src/components/WebBrowser/WebBrowser.js

```diff
@@ -140,10 +140,10 @@ export class WebBrowser extends Component {
             this.logBrowserActions({
               actionType: actionTypes.browserClose,
             });
-            closeWebpage();
+            enableClose.page !== 'blockedPage' && closeWebpage();
           }}
           isSecure={webpage.isSecure}
-          url={webpage.url}
+          url={pageURL ? pageURL : webpage.url}
```

Comment:
What is you reasoning to have `pageUrl` here?

Reply:
I forgot to assign a value to pageURL state. The reason is set the admin url into that url area on browser.

Reply:
Shouldn't this just be `webpage.url`?